chain/ethereum/src/network_indexer/mod.rs

```diff
@@ -52,6 +52,7 @@ impl From<LightEthereumBlock> for Ommer {
             size: block.size,
             mix_hash: block.mix_hash,
             nonce: block.nonce,
+            base_fee_per_gas: block.base_fee_per_gas,
```

Comment:
Dumb question: adding this doesn't affect somehow POI and/or determinism? (I imagine it doesn't)

Reply:
it doesn't, we don't hash web3 structures directly into the PoI, only entities.